crates/uv/tests/pip_compile.rs

```diff
@@ -4560,7 +4560,7 @@ fn no_index_requirements_txt() -> Result<()> {
 
     ----- stderr -----
       × No solution found when resolving dependencies:
-      ╰─▶ Because tqdm was not found in the provided package locations and you require tqdm, we can conclude that the requirements are unsatisfiable.
+      ╰─▶ Because all versions of tqdm were not found in the provided package locations and you require tqdm, we can conclude that the requirements are unsatisfiable.
```

Comment:
Here the formatter methods would need to match on the reason whether it's a whole package or only a version